.travis.yml

```diff
@@ -0,0 +1,19 @@
+sudo: false
+language: python
+python:
+- '3.4'
+- '3.5'
+install:
+- pip install pymodbus3 pyserial
```

Comment:
I used `pip install pymodbus pyserial || pip install pymodbus3 pyserial` in #9.
It is more stable than branch version for the test.